From 2d1ce4879ed64aa1df2cc5779400d47acf341ec4 Mon Sep 17 00:00:00 2001
From: William Gao <wgao1@umbc.edu>
Date: Wed, 6 Sep 2017 09:05:12 -0400
Subject: [PATCH] Added welcome greeting to the kernel log

---
 arch/x86/kernel/setup.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/kernel/setup.c b/arch/x86/kernel/setup.c
index 36646f1..9e42d7f 100644
--- a/arch/x86/kernel/setup.c
+++ b/arch/x86/kernel/setup.c
@@ -881,6 +881,7 @@ void __init setup_arch(char **cmdline_p)
 	 */
 	__flush_tlb_all();
 #else
+	printk("Hello this is willgao's kernel!");
 	printk(KERN_INFO "Command line: %s\n", boot_command_line);
 #endif
 
-- 
2.7.4

